cogs/sessions.py

```diff
@@ -0,0 +1,72 @@
+from asyncio import tasks
+
+from dateutil import parser as timeinput
+from dateutil.parser import ParserError
+from discord import app_commands
+from discord.app_commands import commands
+
+from components.bot import RecruitBot
+from components.config.config_manager import configInstance
+from components.recruitment import recruit
+
+
+# Object representing a recruitment session (user has requested reminders on an interval)
+class Session:
+    ctx: commands.Context
+    interval: int = 30
+    active: bool
+    bot: RecruitBot
+
+    @tasks.loop(seconds=interval)
+    def work(self):
+        if self.active:
+            recruit(self.ctx, self.bot)
+        # else: TODO end the loop if false
+
+
+class SessionsCog(commands.Cog):
+    bot: RecruitBot
+
+    def __init__(self, bot: RecruitBot):
+        self.bot = bot
+        self.sessions = dict()
+
+    @commands.hybrid_command(name="start", with_app_command=True, description="Start a recruitment session")
+    @app_commands.guilds(configInstance.data.guild)
+    async def start(self, ctx: commands.Context, duration: str = None):
+        await ctx.defer()
+
+        # Convert user input to a number of seconds
+        interval = None
+        # Just a number, assume it's seconds
+        if duration.isnumeric():
+            interval = duration
+        # A string, try to parse it
+        else:
+            try:
+                time = timeinput.parse(duration)
+                interval = (time.day * 86400) + (time.hour * 3600) + (time.minute * 60) + (time.second)
```

Comment:
There's python language features for that:
https://stackoverflow.com/questions/4628122/how-to-construct-a-timedelta-object-from-a-simple-string